staff/gerard-celma/cv/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>CV</title>
+    </head>
+    <body>
+        <header>
+            <h1>Waldo Geraldo Faldo</h1>
+            <img src="">
```

Comment:
oups! undefined src and use the alt attribute

Reply:
fixed, thanks!